packages/front-end/components/Metrics/MetricTooltipBody.tsx

```diff
@@ -69,7 +70,11 @@ const MetricTooltipBody = ({
         .map(({ label, body }, index) => (
           <div key={`metricInfo${index}`}>
             <strong>{`${label}: `}</strong>
-            <span className="font-weight-normal">{body}</span>
+            {label === "Description" ? (
```

Comment:
It's possible to simplify this and just always render the `Markdown` component, but that does mean we'll need to cast numbers to strings within the `Markdown` component in order to provide `dangerouslySetInnerHTML` with a string.

Reply:
Just change the `MetricInfo` interface to have a boolean `markdown` property and only set it to `true` for the description.  That will make this code much more maintainable.